index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>microverse_trails_with_milenitic</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="body">
+        <div id="body-wrapper">
+    <div id="main">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+    </div>
+    <div id="form">
+        <form id="survey-form">
```

Comment:
I would suggest you wrap this with a section tag. A section tag generic semantic element, that is used to combine elements related together into a discrete unit. kindly refer back to the [curriculum-html-css](https://github.com/microverseinc/curriculum-html-css) for clarity🤗